src/restSamples.ts

```diff
@@ -0,0 +1,222 @@
+'use strict';
+
+import * as vscode from 'vscode';
+import { Swagger } from './swagger';
+import { PlaybookManager } from './playbookManager';
+import * as utilities from './utilities';
+var path = require("path");
+var fs = require('fs');
+var pm = new PlaybookManager();
+
+export class RestSamples {
+    protected _outputChannel: vscode.OutputChannel;
+
+    constructor(outputChannel: vscode.OutputChannel) {
+        this._outputChannel = outputChannel;
+    }
+
+    public displayMenu() {
+        this.createRestApiCall();
+    }
+
+    // - query list of api groups (compute, etc...)
+    // - user selects api group
+    public createRestApiCall() {
+        let __this = this;
+
+        this.getSpecificationLocation(function(specLocation) {
+            __this.queryApiGroups(specLocation, function (groups) {
+                if (groups != null) {
+                    vscode.window.showQuickPick(groups).then(selection => {
+                        __this.selectOperation(specLocation + "/specification/" + selection);
+                    });
+                } 
+            })
+        })
+    }
+
+    public selectOperation(path: string) {
+
+        let operations = this.queryAll(path);
+        let __this = this;
+
+
+        let items = [];
+
+        for (var key in operations) {
+            items.push(operations[key]);
+        }
+
+        vscode.window.showQuickPick(items).then(selection => {
+            // the user canceled the selection
+            if (!selection) return;
+
+            items = [];
+
+            for (var f in selection['files']) {
+                let swagger = require(selection['files'][f]);
+                if (swagger != null) {
+                    let xpath =  selection['files'][f].split('/').slice(0, -1).join('/');
+                    let swaggerHandler = new Swagger(swagger, selection['files'][f].split('/').slice(0, -1).join('/'));
+                    let examples: string[] = swaggerHandler.getExampleNames(selection['path'], selection['method']);
+                    let apiVersion = xpath.split('/').slice(-1)[0];
+
+                    examples.forEach(function(s, i, a) {
+                        items.push({
+                            'label': apiVersion + ' - Sample: ' + s.split('/').pop(),
```

Comment:
as we discussed, remove the word 'Sample"here

Reply:
yes, sorry, forgot....